### DISCUSSION TITLE
Friendly Path

### DISCUSSION THREAD

**Discussion by tebosoftware** [HashLoad/horse#238] [General]
Good afternoon, is there a way to add a friendly path to the methods looking like this:
/client/1
where would the 1 be converted to a parameter called id?

**Comment by viniciussanchez** [comment]
```pascal
procedure MyCallBack();
var
  LId: string;
begin
  LId := Req.Params['id'];
end;

THorse.Get('/client/:id', MyCallBack);
```

http://localhost:9000/client/1

**Reply by tebosoftware** [reply]
Thanks
